h2o-hadoop/h2o-mapreduce-generic/build.gradle

```diff
@@ -16,7 +16,8 @@ compileJava {
 
 dependencies {
   compile('org.apache.hadoop:hadoop-client:' + hadoopMavenArtifactVersion)
-  compile "org.eclipse.jetty.aggregate:jetty-all-server:8.1.17.v20150415"
+  compile "org.eclipse.jetty:jetty-proxy:${jettyVersion}"
```

Comment:
@mmalohlava I'm not sure about this one. Should all required dependencies be here because of they are supposed to go into driver uber-jar ?